src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -1111,10 +1111,13 @@ export type Level = 'error' | 'fatal' | 'info' | 'warning' | 'sample';
 export type Meta = {
   chunks: Array<Chunks>;
   len: number;
-  rem: Array<Array<string | number>>;
-  err: Array<any>;
+  rem: Array<MetaRemark>;
+  err: Array<MetaError>;
 };
 
+export type MetaError = [string, any];
```

Comment:
interesting... I know what you want to achieve but not sure if we should do it like this